app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala

```diff
@@ -318,8 +320,17 @@ class SubsidyController @Inject() (
         addClaimDate <- subsidyJourney.claimDate.value.toContext
         previous = subsidyJourney.previous
       } yield {
-        val form = subsidyJourney.claimAmount.value.fold(claimAmountForm)(claimAmountForm.fill)
-        Ok(addClaimAmountPage(form, previous, addClaimDate.year, addClaimDate.month))
+        val form = subsidyJourney.claimAmount.value.fold(claimAmountForm) { ca =>
```

Comment:
Can we rename ca? Intellij will always fill that stuff in on rename and then autocomplete works on camel case so you can autocomplete from ca